.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+<!-- What does this PR do? Please list any issue that these changes address and how you have gone about implementing the changes -->
+
+## Checklist
+
+I confirm that I have completed the following checks:
+
+**Testing**
+- [ ] If I have changed any of the Python or Fortran code, I have rebuilt and run the tests locally and they have passed.
+- [ ] I have addressed any differences in the regression tests caused by this pull request in the comments.
+- [ ] I have added new tests where appropriate for the changes I have made.
+- [ ] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
+
+**Quality**
+- [ ] I have ensured my changes are formatted correctly and conform to PROCESS' quality standards.
+- [ ] If I have made documentation changes, I have checked they render correctly.
+
+**Miscellaneous**
+- [ ] I have created a changelog entry for my change, if appropriate.
```

Comment:
Seeing as hardly anyone has used the changelog tool, is this still worth doing? Perhaps one to think about.

Reply:
I know a lot of codes don't really have a CHANGELOG.md anymore and instead use releases to document their changes (which is automatically picked up by GitHub). If we are going to have a changelog we need to enforce its use now we are OS.

But I would be for removing the changelog and keeping the information in the releases

Reply:
I agree. I've created #2930 to deal with this issue explicitly.